test/VaultFactory.t.sol

```diff
@@ -24,7 +26,7 @@ contract VaultFactoryTest is Test {
 
     function setUp() public virtual {
         Deploy deployment = new Deploy();
-        (vaultFactory, stakeManager, deploymentConfig) = deployment.run();
+        (vaultFactory, stakeManager, migrationStakeManager, deploymentConfig) = deployment.run();
```

Comment:
I've already mentioned above that this deployment script should not return a `migrationStakeManager` in the first place, but regardless, given that it's also not used anywhere here, we also don't need to add it to the test contract.